docs/content/config.md

```diff
@@ -222,6 +222,7 @@ See also: [Feature Description]({{% relref "/features#access-menu" %}}), [keybin
 | Speak Action Bar Messages              | true          | Whether to speak the messages updated in [action bar](https://minecraft.wiki/w/Commands/title), useful when you're in modded multiplayer servers                          |
 | Only Speak Action Bar Updates          | false         | Only speak changed part of action bar message when the message is partially updated, useful for some mods like Hypixel                                                    |
 | Speak Harvest Of Fishing               | true          | Whether to speak the harvest of fishing                                                                                                                                   |
+| Always Speak Picked Up Items           | false         | Wether to speak any items you pick up (this will also enable speaking of fishing harvests)                                                                                |
```

Comment:
I don't think this has anything to do with a castrated ram.
```suggestion
| Always Speak Picked Up Items           | false         | Whether to speak any items you pick up (this will also enable speaking of fishing harvests)                                                                                |
```